src/transformers/tokenization_utils_fast.py

```diff
@@ -870,3 +877,53 @@ def train_new_from_iterator(
             kwargs["additional_special_tokens"] = additional_special_tokens
 
         return self.__class__(tokenizer_object=tokenizer, **kwargs)
+
+    def update_post_processor(self):
```

Comment:
Cool, this actually overwrites the original post_processor, but that is what we want!